src/test/java/de/tum/in/www1/artemis/metis/AnswerPostIntegrationTest.java

```diff
@@ -36,6 +40,8 @@
 
     private Long courseId;
 
+    private int maxPostsPerPage = 20;
```

Comment:
```suggestion
    private static final int MAX_POSTS_PER_PAGE = 20;
```
This value seems like a constant so it may be better to make it explicit. There are similar usages like that below.

Reply:
Implemented, thanks! 👍🏻